gapic-generator-cloud/Rakefile

```diff
@@ -46,6 +46,25 @@ namespace :gen do
   end
 end
 
+desc "Start an interactive shell."
+task :console do
+  require "irb"
+  require "irb/completion"
+  require "pp"
+
+  $LOAD_PATH.unshift "lib"
+
+  require "google/gapic/schema"
+  def schema service
+    proto_input = File.binread "proto_input/#{service}.bin"
```

Comment:
This looks like a nice way to explore the schema object. Could we take a proto as input here and compile it or do you think that's overkill? Is there another reason to want to use a `.bin` file?

Reply:
Right now there is still some configuration specific to each service used to build the binary file in the shared Rakefile. I suppose we could put that somewhere where it can be easily used across all the directories. But I think it is easier to load a file and inspect it.

Reply:
That makes sense for now. I could see something like this being pretty helpful to help get someone started, but let's not complicate it now while the config is still in motion.